pkg/webui/console/views/device-add/wizard/network-settings-form/index.js

```diff
@@ -93,6 +97,25 @@ const NetworkSettingsForm = props => {
     setResetsFCnt(checked)
   }, [])
 
+  const [deviceClasses, setDeviceClasses] = React.useState([DEVICE_CLASSES.CLASS_A])
+  const handleDeviceClassChange = React.useCallback(evt => {
+    const { checked, name } = evt.target
+
+    let cls
```

Comment:
Exclusion of those 2 characters must have really sped up the development /s
```suggestion
    let class
```

Reply:
`class` is a keyword in JavaScript.

Reply:
1. yes
2. `class` is reserved in javascript and can be used only to define a class
3. we already use `cls` in many places throughout the codebase

Reply:
If shadowing is not allowed, then that really sucks!
Anyway, just want to point out that this really stands out and is not obvious for a person with not much context about frontend. I wish there was a clearer way to denote this, but I can't think of any.